assets/multiqc_config.yaml

```diff
@@ -2,6 +2,198 @@ report_comment: >
     This report has been generated by the <a href="https://github.com/nf-core/clipseq" target="_blank">nf-core/clipseq</a>
     analysis pipeline. For information about how to interpret these results, please see the
     <a href="https://github.com/nf-core/clipseq" target="_blank">documentation</a>.
+
+data_format: 'yaml'
+
+run_modules:
+    - custom_content
+    - fastqc
+    - cutadapt
+    - bowtie2
+    - star
+    - preseq
+    - rseqc
+
+exclude_modules:
+    - 'general_stats'
+
+extra_fn_clean_exts:
+    - type: remove
+      pattern: .premap
+      module: bowtie2
+    - type: remove
+      pattern: _reads
+      module: fastqc
+
+parent_id: mapping
+parent_description: 'This section contains mapping metrics'
+
+parent_id: dedup
+parent_description: 'This section contains PCR deduplication metrics'
+
+parent_id: crosslinks
+parent_description: 'This section contains crosslink metrics'
+
+parent_id: peaks
+parent_description: 'This section contains peak metrics'
```

Comment:
I think that you can delete these lines. They're just overwriting each other and they don't have any function on this top level anyway (you added them in the right place already, in the `custom_data` section underneath).

Reply:
resolved: fdad814